examples/https_client/https_client.cc

```diff
@@ -0,0 +1,184 @@
+// Create an https client test program
```

Comment:
please move it to examples folder and rename it to https_client_cli

Reply:
moved